PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: WorMzy Tykashi <wormzy.tykashi@gmail.com>
```

Comment:
I don't think this file belongs in the PR.

Reply:
Ah damn, I made a mess